### PR TITLE
12/n - DEVREL-58 : Cover Invoice Line APIs

### PR DESCRIPTION
**Pull request by Aman-Aalam**
# Fixes #

* Covered Invoice Line APIs
* Fixed a typo in exception message of Invoice.Update
* Reorganized some variable initialization to group similar actions together

### Checklist

- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, create a GitHub Issue in this repository.

### CODE REVIEW

File: trolley/invoice_line_gateway.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    """
+
+    def __init__(self, gateway, config):
+        self.gateway = gateway
+        self.config = config
+
+    """ Creates a new InvoiceLine. Only provide the line item in the body, and provide invoiceId separately.  """
+    def create(self, invoice_id, body,):
+        if body is invoice_id:
+            raise InvalidFieldException("Invoice ID cannot be None.")
```
Comment (julie-trolley) [inline trolley/invoice_line_gateway.py:19]: Do you mean "if invoice_id is None:"?

Comment (Aman-Aalam) [reply trolley/invoice_line_gateway.py:19]: Oh, good catch!

Comment (Aman-Aalam) [reply trolley/invoice_line_gateway.py:19]: @julie-trolley  I have fixed it here: https://github.com/trolley/python-sdk/commit/1fb30893e79c48d6c54590a6671cfb33144092a8

but somehow it doesn't show up here.

Comment (julie-trolley) [reply trolley/invoice_line_gateway.py:19]: saw it, maybe there's delay:)

**Review by julie-trolley — APPROVED** [review]

**Pull request merged**